### PR TITLE
Update tests/examples to v1

### PR DESCRIPTION
**Pull request by nikhiljindal**
Updating tests/examples to v1, so that they dont break once we stop supporting v1beta3.
Ref: #10501

Forked out changes from https://github.com/GoogleCloudPlatform/kubernetes/pull/10525

### CODE REVIEW

**Comment by nikhiljindal** [comment]
cc @bgrant0607 @lavalamp

**Comment by bgrant0607** [comment]
LGTM

**Comment by bgrant0607** [comment]
@k8s-bot test this please

**Comment by nikhiljindal** [comment]
Rebased!

File: pkg/api/latest/latest.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ var userResources = []string{"rc", "svc", "pods", "pvc"}
 func init() {
 	// Use the first API version in the list of registered versions as the latest.
 	Version = registered.RegisteredVersions[0]
+	OldestVersion = registered.RegisteredVersions[len(registered.RegisteredVersions)-1]
```
Comment (lavalamp) [inline pkg/api/latest/latest.go]: This looks wrong-- aren't versions registered in init() functions? if so, the order is arbitrary?

Comment (nikhiljindal) [inline pkg/api/latest/latest.go]: There is now api/registered that registers api versions in the desired order: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/registered/registered.go#L41

Comment (lavalamp) [inline pkg/api/latest/latest.go]: Oh, right. Thanks!

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**